themes/hello-friend-ng/assets/scss/_main.scss

```diff
@@ -26,8 +26,8 @@ body {
   -webkit-text-size-adjust: 100%;
 
   display: flex;
-  min-height: 100vh;
   flex-direction: column;
+  height: 100%;
 
```

Comment:
I'm sorry to request changes again but can you add that `min-height: 100vh;` again? Without it it doesn't align with the bottom of the viewport:
![Screenshot from 2021-11-22 13-07-39](https://user-images.githubusercontent.com/31856436/142935844-947515a2-e650-4a31-9612-c30b526544ea.png)